wear/src/main/java/io/homeassistant/companion/android/home/views/EntityUi.kt

```diff
@@ -57,7 +60,24 @@ fun EntityUi(
                 Text(
                     text = friendlyName,
                     maxLines = 2,
-                    overflow = TextOverflow.Ellipsis
+                    overflow = TextOverflow.Ellipsis,
+                    modifier = Modifier.pointerInput(Unit) {
```

Comment:
Long press now only works on the entity's label, I think this should be on the chip in case you have a short label. Is that possible? Otherwise the label should at least fill the width of the chip instead.

Reply:
Unfortunately we cannot add a long press on the chip itself, since it already has a gesture handler and we can apparently not override it. So on the label seems the only possibility. You could check the linked issue for more info 🙂

I will make the label fill the width though, good suggestion!

Reply:
I didn't check all comments in the issue before commenting 😳 and I've actually looked at the source code for Chip before and was aware of the limitation, so I really should have noticed this before commenting. 
The comment was because I couldn't reliably trigger the details screen, so changing the label width will help!